malasakit-django/pcari/templates/qualitative-questions.html

```diff
@@ -47,11 +47,13 @@
     {% for qid, prompt in questions %}
       <li>
         <p class="prompt">{{ prompt }}</p>
-        <textarea id="comment-{{ qid }}" class="comment"></textarea>
+        <center>
+          <textarea class="comment"></textarea>
```

Comment:
This breaks the JavaScript. Please put back `id` set to `comment-<qid>` (this is how we can tell which qualitative question this comment is linked to).